content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,67 @@
+# Internal link checking tool
+
+You can use [htmltest](https://github.com/wjdp/htmltest) to check for broken links in [`/content/en/`](https://git.k8s.io/website/content/en/). This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What it does and doesn't check
+
+The link checker scans generated HTML files, not raw markdown. You can see the full configuration in [`.htmltest.yml`](https://git.k8s.io/website/.htmltest.yml).
+
+The link checker scans the following:
+
+- All content generated from Markdown in [`/content/en/docs`](https://git.k8s.io/website/content/en/docs/) directory, excluding:
+  - Generated API references
```

Comment:
Line 12, `Generated API references` is referring to the individual component tool reference pages (which are .md files)?

Reply:
What's the difference between scanning `All content generated` and `All internal links`?

Reply:
@kbhawkey line 12: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/ & older versions of the API reference aren't scanned. (Specifically, anything that lands in the `reference/generated/kubernetes-api` directory in the site's final build is ignored)